arabot/utils/formatting.py

```diff
@@ -42,4 +43,8 @@ def repchars(s: str, chars: str, rep: str = "") -> str:
     return _re.sub(f"[{_re.escape(chars)}]", rep, s)
 
 
+def humanjoin(s: _Iterable[str], /):
+    return " and ".join(", ".join(s).rsplit(", ", 1))
```

Comment:
Turns `['foo', 'bar', 'baz']` into `'foo, bar and baz'`. Forgot the return value type hint, will add it later.

Reply:
Fixed in e85a76c